yarn.lock

```diff
@@ -6853,14 +6853,6 @@ ajv-keywords@^3.5.2:
   resolved "https://registry.yarnpkg.com/ajv-keywords/-/ajv-keywords-3.5.2.tgz#31f29da5ab6e00d1c2d329acf7b5929614d5014d"
   integrity sha512-5p6WTN0DdTGVQk6VjcEju19IgaHudalcfabD7yhDGeA6bcQnmL+CpveLJq/3hvfwd1aof6L386Ougkx6RfyMIQ==
 
-ajv@^4.9.1:
```

Comment:
Interesting, why do we have so many changes in the yarn.lock file? 🤔

Reply:
Honestly, I don’t know, I just ran `kbn run bootstrap`